handlers/UpdateHandler__.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace YesWiki\Shop;
+
+use YesWiki\Core\YesWikiHandler;
+use YesWiki\Security\Controller\SecurityController;
+
+class UpdateHandler__ extends YesWikiHandler
+{
+    public function run()
+    {
+        if ($this->getService(SecurityController::class)->isWikiHibernated()) {
+            throw new \Exception(_t('WIKI_IN_HIBERNATION'));
+        };
+        if (!$this->wiki->UserIsAdmin()) {
+            return null;
+        }
+
+        $output = $this->wiki->render('@shop/update-handler.twig', []);
```

Comment:
en fait, je n'utiliserai pas un rendu à cet endroit.
Je verrai plutôt quelque chose du type:
 - récupération du paramètre `shop['forms']['products']`
 - pour chaque identifiant de ce tableau (ou chaîne de caractères séparées par des virgules à `explode`)
   - utiliser la fonction `getFormOrCreate` qui aura été importée depuis `GestionShopHandler`
    - pour les formulaires créés, utiliser le template fournit dans `setup/forms/`
   - si vide : ne rien faire
  - afficher un message du genre ` $output .= 'ℹ️ Creating the <em>form</em> $id<br />';`
  
...

` $output .= '✅ Done !<br />';`

ou 
`
                $output .= "! Not possible to ...<br />";`